### ISSUE TITLE
The game crashes when trying to generate beehives on aspen trees

### ISSUE THREAD

**Issue by lumiscosity** [great-big-world/great-big-world#21]
[Log of the issue](https://mclo.gs/R8nIZc7)
Occasionally, the placement of the trees_aspen_mixed feature fails, though this probably happens for all other forms of aspen tree generation, causing the server to crash. Reading the logs, this seems to be caused by beehive generation failing; sure enough, modifying the green and yellow aspen tree configured features fixes the crash.

**Comment by jack-zisa** [comment]
Yes, looks like this issue is the result of some other mod. I cannot recreate it with just Great Big World installed. I'll do some more research into what mod might be causing it.

**Comment by lumiscosity** [comment]
decided to sit down and properly debug this, and i ended up getting the crash to happen with no other mod installed. seed 4461729434966573269 on single biomes aspen forest, move towards +X coordinates and the game will crash [(updated log with no other mods)](https://mclo.gs/pTBxX32)
